tests/api/CommentsApiTest.php

```
@@ -308,4 +421,67 @@ protected function createTestComment(): int
+    /**
+     * 단순 3단 트리
+     *
+     * @return array{root: CommentEntity, child: CommentEntity, grandchild: CommentEntity}
+     */
+    protected function createSimpleTree(int $postId): array
+    {
+        $fab = new Fabricator(CommentModel::class);
+        $fab->setOverrides(['post_id' => $postId, 'parent_id' => null, 'state' => CommentState::APPROVED->value]);
+        $root = $fab->create();
+
+        $fab->setOverrides(['post_id' => $postId, 'parent_id' => $root->id, 'state' => CommentState::APPROVED->value]);
```

PHPStan (2.1.54)

[error] Cannot access property $id on array|object.


(property.nonObject)